library/std/src/io/error.rs

```diff
@@ -307,23 +314,24 @@ pub enum ErrorKind {
 impl ErrorKind {
     pub(crate) fn as_str(&self) -> &'static str {
         use ErrorKind::*;
+        // Strictly alphabetical, please.  (Sadly rustfmt cannot do this yet.)
```

Comment:
If you think it's warranted to check this automatically, https://crates.io/crates/remain

Reply:
Hah thanks :-).  That would be nice, indeed.

However, I don't think this is used in rust-lang/rust already.  I'm not at all sure what the rules/culture/whatever is for dependencies.  If it's just for this one instance, perhaps it's not worth it?

Reply:
I would strongly prefer if no proc-macros are used by libstd or it's dependencies.